z3tracer/src/model.rs

```diff
@@ -96,8 +95,9 @@ impl Model {
     where
         R: std::io::BufRead,
     {
+        let parser_config = ParserConfig { ignore_invalid_lines: true };
```

Comment:
FYI `let config = ParserConfig { .. }` is also totally fine with me (but I do generally avoid abbreviations like `let pc =`)